constellations/orion/src/tests_integration/user_afh_integration_test.go

```diff
@@ -0,0 +1,193 @@
+package tests_integration
+
+import (
+	"encoding/json"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/ahsu1230/mathnavigatorSite/constellations/orion/src/domains"
+	"github.com/ahsu1230/mathnavigatorSite/constellations/orion/src/tests_integration/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+// Test: Create 3 UserAfhs. Then get by userId
+func Test_GetUserAfhsByUserId(t *testing.T) {
+	createAllUserAfhs(t)
+
+	// Call GetUserAfhsByUserId()
+	recorder := utils.SendHttpRequest(t, http.MethodGet, "/api/userAfhs/userAfhs/user/2", nil)
+
+	// Validate results
+	assert.EqualValues(t, http.StatusOK, recorder.Code)
+	var userAfhs []domains.UserAfh
+	if err := json.Unmarshal(recorder.Body.Bytes(), &userAfhs); err != nil {
+		t.Errorf("unexpected error: %v\n", err)
+	}
+
+	assert.EqualValues(t, 2, userAfhs[0].UserId)
+	assert.EqualValues(t, 2, userAfhs[0].AfhId)
+
+	resetUserAfhTables(t)
+}
+
+// Create 3 UserAfhs. Then update one. Then get by afhId
+func Test_GetUserAfhsByAfhId(t *testing.T) {
+	createAllUserAfhs(t)
+
+	// Update
+	updatedUserAfh := createUserAfh(2, 1)
+	updatedBod := utils.CreateJsonBody(&updatedUserAfh)
+	recorder2 := utils.SendHttpRequest(t, http.MethodPost, "/api/userAfhs/userAfh/1", updatedBod)
+	assert.EqualValues(t, http.StatusOK, recorder2.Code)
```

Comment:
I'm not sure why this is returning with "Error #01: sql: expected 3 arguments, got 4", especially since I pretty much did the same thing when I was testing the endpoints in my user_afh_controller_test.go file. Otherwise, the code works fine.